Audible-Altimeter-Embedded/src/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+set(LIB_NAME Audible-Altimeter-Embedded)
+
+set(include_dir ${CMAKE_CURRENT_SOURCE_DIR}/../include)
+set(sources
+    Tiny2350.cpp
+    )
+set(includes
+    ${include_dir}/Audible-Altimeter-Embedded/ITiny2350.hpp
+    ${include_dir}/Audible-Altimeter-Embedded/Tiny2350.hpp
+    )
+
+if (${IS_PICO})
+    add_library(${LIB_NAME} STATIC ${sources} ${includes})
+    add_library(${LIB_NAME}::Audible-Altimeter-Embedded ALIAS ${LIB_NAME})
+    target_link_libraries(${LIB_NAME} PUBLIC
+        hardware_i2c
```

Comment:
just a heads up - i had to include Audible-Altimeter::Audible-Altimeter here to get audio to work. The header linking was working just fine but the function implementations couldn't be found. does that mean i was doing something wrong?

Reply:
Hmm, yeah the embedded library should most likely not have a link between the Audible-Altimeter library. I'll try to look for that in your PR and see what is happening!